app/javascript/components/mentoring/discussion/PreviousSessionsLink.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+export const PreviousSessionsLink = ({
```

Comment:
The component seems to indicate that this is a link, but I don't see a link?

Reply:
We don't have the modal to display after the link is clicked yet. I just wrapped it in this component in preparation.